### ISSUE TITLE
Improvement: add new tool to convert any string to QR-code in shell

### ISSUE THREAD

**Issue by chubin** [alexanderepstein/Bash-Snippets#19]
You can convert any string to a QR-code without leaving your shell/editor 
using curl and and qrenco.de:

```
$ printf "two\nlines" | curl -F-=\<- qrenco.de
$ curl qrenco.de/this                  
```

You can use any of these two forms (this first one supports any symbols + multilne text; the second one is shorter).

You could add a new script, say `qrencode` that will encode text passed as its argument,
or read the text from stdin and convert it using qrenco.de

Something like this:

```
if [ -z "$1" ]; then 
  curl -F-=\<- qrenco.de
else
  printf "$*" | curl -F-=\<- qrenco.de
fi
```

What is it good for? There are many good usecases, my favourite one is to quickly bring up text from your console on your smartphone.

**Comment by linyostorovovoltos** [comment]
Sure! I'm working on this right now, but I don't know if it'll be accepted. Thanks for the infos!

**Comment by alexanderepstein** [comment]
@linyostorovovoltos will definitely be accepted if I agree with the method and I can modify the for similar to the way you set up ytview I have been busy spending the time I get to work on this on cloudup because that is the most useful tool to me

**Comment by alexanderepstein** [comment]
Released as of version 1.10.0

**Comment by chubin** [comment]
@linyostorovovoltos I've commented it in the code directly, but I repeat it here also:
don't do `read a; echo $a | curl -F...` just use `curl -F...` instead
